### ISSUE TITLE
the plugin broken in reanimated 3.4.2

### ISSUE THREAD

**Issue by YOEL311** [wcandillon/eslint-plugin-reanimated#37]
I use your plugin 
all work as well, but after I upgrade the version of the ``react-native-reanimated`` library to the ``3.4.2`` version 
I got a false warning 

````
scrollTo is not a worklet. Use runOnJS instead.eslint
[reanimated/js-function-in-worklet](https://github.com/wcandillon/eslint-plugin-reanimated/blob/master/docs/js-function-in-worklet.md)
````

![image](https://github.com/software-mansion/react-native-reanimated/assets/39925212/7e8f7c12-6738-4a4d-8165-a614accba502)

![image](https://github.com/software-mansion/react-native-reanimated/assets/39925212/82c0260e-feb0-49f1-9b5f-3f081dbb9f21)

![image](https://github.com/software-mansion/react-native-reanimated/assets/39925212/7250eaee-46f8-4454-88a0-56791095b709)

the code:
````
 const animationScrollToFlat = (
    ref: AnimatedRef<Animated.FlatList<string>>,
  ) => {
    'worklet';
    scrollTo(ref, 0, Math.random() * 300, true);
  };

  const animationScrollToSection = (
    ref: AnimatedRef<
      React.Component<
        AnimateProps<SectionListProps<string, DefaultSectionT>>,
        unknown,
        unknown
      >
    >,
  ) => {
    'worklet';
    scrollTo(ref, 0, Math.random() * 300, true);
    
    
  const backgroundColorAnimStyle = useAnimatedStyle(() => {
    return {
      backgroundColor: interpolateColor(
        sharedAnimation.value,
        [0, 1],
        ['#00001', '#12345'],
      ),
    };
  }, []);


````

you can reproduce this warning in this repo

https://github.com/YOEL311/bugReanimatedRecovery

i also attach the diff between ``3.4.1`` to ``3.4.2``

https://github.com/software-mansion/react-native-reanimated/compare/%40piaskowyk/3.4.1...%40piaskowyk/3.4.2

I have corresponded with the reanimated team about this
https://github.com/software-mansion/react-native-reanimated/issues/4873#issuecomment-1680733874

**Comment by wcandillon** [comment]
Thanks for reporting this, a fix will be published in a few minutes.

**Comment by YOEL311** [comment]
Thanks
